### PR TITLE
Edit ug

### PR DESCRIPTION
**Pull request by Superbestron**
Made it clear that:
1. 'AddTaskCommand' requires added elderly name to be first registered in NurseyBook.
2. 'EditTaskCommand' requires new edited elderly name to be first registered in NurseyBook.
3. 'EditCommand' updates all tasks containing the name of the elderly to edit with the new elderly name.
4. 'DeleteCommand' deletes all occurrences of the name of the elderly for tasks that contain the specfied elderly name in the command.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -493,19 +495,26 @@ e.g. Date of task is `2022-01-01` with `daily` recurrence. On `2022-01-03`, the
 
 </div>
 
-Example:
-* `viewTasks` followed by `editTask 1 date/2022-01-30` changes the date of the 1st task shown by NurseyBook to 30 January 2022.
+<div markdown="block" class="alert alert-warning">
+
+:exclamation: **Caution**
+* If you want to edit the elderlies' names of the task, you need to first make sure all the elderlies' names are already registered in your elderly database.
```
Comment (CraveToCode) [inline docs/UserGuide.md:501]: Perhaps rephrasing to `If you want to edit the names of any elderlies linked to a task, ...` would be better.

**Review by CraveToCode — CHANGES_REQUESTED** [review]
UG changes look good to me! Just a small suggestion in phrasing for better clarity.

File: docs/UserGuide.md (hunk 1 above)
Comment (Superbestron) [reply docs/UserGuide.md]: Made the changes!

**Review by CraveToCode — APPROVED** [review]
Looks good to go!

**Pull request merged**